Adding iframe case to troubleshooting guide (#10869)
DataDog/documentation: content/en/synthetics/troubleshooting/_index.md

After downloading the [Datadog extension][2], you are unable to see your website in the iframe on the right side of your Browser test's recorder and the iframe displays `Your website does not support being loaded through an iframe.`. This could mean that your application has some settings preventing it from being opened in an iframe. If that is the case, try opening your website in a pop up by clicking **Open in Popup** to record your journey.

#### Some of my applications load in the iframe but some do not

> This means your applications and environemnts most likely have different restrictions causing some of them to be able to be visualized in an iframe, and some others not to.

Reviewer: ```suggestion
This means your applications and environments have different restrictions, which causes some of them to be visualized in an iframe while the others are not viewable.
```
